homeassistant/components/flux_led/translations/en.json

```diff
@@ -27,8 +27,7 @@
                 "data": {
                     "custom_effect_colors": "Custom Effect: List of 1 to 16 [R,G,B] colors. Example: [255,0,255],[60,128,0]",
                     "custom_effect_speed_pct": "Custom Effect: Speed in percents for the effect that switch colors.",
-                    "custom_effect_transition": "Custom Effect: Type of transition between the colors.",
-                    "mode": "The chosen brightness mode."
```

Comment:
This is vestigial from when the integration couldn't figure it out automatically.